content/doc/book/blueocean/creating-pipelines.adoc

```diff
@@ -64,33 +63,37 @@ repository in the *Repository URL* field.
 image:blueocean/creating-pipelines/connect-to-a-git-repository.png['Connect to a
 Git repository',width=70%]
 
-You now need to specify a <<local-repository,local>> or a
-<<remote-repository,remote>> repository from which to build your Pipeline
+You now need to specify a link:#local-repository[local] or a
+link:#remote-repository[remote] repository from which to build your Pipeline
 project.
 
 
 ==== Local repository
 
 If your URL is a local directory path (e.g. beginning with a forward slash `/`
-such as `/home/cloned-git-repositories/my-git-repository.git`), you can proceed
-to click the *Create Pipeline* button.
+such as `/home/cloned-git-repos/my-git-repo.git`), you can proceed to click the
+*Create Pipeline* button.
 
 Blue Ocean will then scan your local repository's branches for a `Jenkinsfile`
 and will commence a Pipeline run for each branch containing a `Jenkinsfile`. If
 Blue Ocean cannot find any `Jenkinsfile`, you will be prompted to begin creating
-one through the <<pipeline-editor#,Blue Ocean Pipeline editor>>.
+one through the <<pipeline-editor#,Pipeline editor>>.
 
 
 ==== Remote repository
 
+Since the Pipeline editor saves edited Pipelines to Git repositories as
+`Jenkinsfile`&zwj;s, Blue Ocean only supports connections to remote Git
+repositories over the SSH protocol.
+
 If your URL is for a remote Git repository, then as soon as you begin typing
 the URL, starting with either:
 
 * `ssh://` - e.g.
-  `ssh://gituser@server-url/git-server-repositories/my-git-repository.git` +
+  `ssh://gituser@git-server-url/git-server-repos-group/my-git-repo.git` +
   or
 * `user@host:path/to/git/repo.git` - e.g.
-  `gituser@server-url:git-server-repositories/my-git-repository.git`,
+  `gituser@git-server-url:git-server-repos-group/my-git-repo.git`,
```

Comment:
Ditto about shortening these. "Repositories" is a long word and I wanted to specify that the `server-url` was a `git-server-url`.